README.md

```diff
@@ -3,12 +3,15 @@
 ## Run with Docker
 To run the web portal using the Dockerfile:  
 
```

Comment:
neglected to include that a Wolfram app id is necessary for the web portal to run